MAUI/DataGrid/selection.md

```diff
@@ -0,0 +1,547 @@
+<td>
+Moves the current cell to the last cell of the current row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>PageDown</kbd>
+</td>
+<td>
+The SfDataGrid will be scrolled to next set of rows that are not displayed in view, including the row that are partially displayed and the current cell is set to last row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>PageUp</kbd>
+</td>
+<td>
+The SfDataGrid will be scrolled to previous set of rows that are not displayed in view, including the row that are partially displayed and the current cell is set to the first row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Tab</kbd>
+</td>
+<td>
+Moves the current cell to next to the active current cell. If the active current cell is the last cell of the current row, the focus will moved to first cell of the row next to the current row.If the active current cell is the last cell of the last row, the focus will be moved to next control in the tab order of the parent container.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Shift</kbd> + <kbd>Tab</kbd>
+</td>
+<td>
+Moves the current cell to previous to the active current cell. If the active current cell is the first cell of the current row, the current cell will moved to last cell of the row previous to the current row.If the active current cell is the first cell of the first row, the focus will be moved to previous control in the tab order of the parent container.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>DownArrow</kbd>
+</td>
+<td>
+Moves the current cell to the current column of the last row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>UpArrow</kbd>
+</td>
+<td>
+Moves the current cell to the current column of the first row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>Home</kbd>
+</td>
+<td>
+Moves the current cell to the first cell of the first row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>End</kbd>
+</td>
+<td>
+Moves the current cell to the last cell of the last row.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Enter</kbd>
+</td>
+<td>
+If the active current cell is in edit mode, the changes will committed and moves the current cell to below the active current cell. If the active current cell is in last row, commits changes only and retains in the same cell.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>A</kbd>
+</td>
+<td>
+All rows or cells will be selected.
+</td>
+</tr>
+</table>
+
+## Move Current Cell
+The [CurrentCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CurrentCell) can be moved to a particular [RowColumnIndex] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex.html) by using the [SfDataGrid.MoveCurrentCellTo()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_MoveCurrentCellTo_Syncfusion_Maui_GridCommon_ScrollAxis_RowColumnIndex_System_Boolean_) method. This method is not applicable when the `SfDataGrid.SelectionMode` is None or `NavigationMode` is Row.
```

Comment:
The [CurrentCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CurrentCell) can be moved to a particular [RowColumnIndex] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex.html) by using the [SfDataGrid.MoveCurrentCellTo()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_MoveCurrentCellTo_Syncfusion_Maui_GridCommon_ScrollAxis_RowColumnIndex_System_Boolean_) method. This method is not applicable when the `SfDataGrid.SelectionMode` is None or the `NavigationMode` is Row.